### ISSUE TITLE
Books on floor disappear with build 6589

### ISSUE THREAD

**Issue by Zetsukaze** [CleverRaven/Cataclysm-DDA#21421]
### Expected Behavior
Books left on the floor remain there
### Actual behavior
After crafting/sleeping a bit near the books, they disappear.
### Steps to reproduce the behavior
Unsure, just notice that without leaving the area while crafting, the books on the floor are gone.

[Thrix.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/1149612/Thrix.zip)

**Comment by Zetsukaze** [comment]
Happens when there is an empty shopping cart over the tile where the books are dropped, if I suspect correctly. I have two piles of books, and only one disappears. Happened twice already.

**Comment by busterbogheart** [comment]
Can you clarify the steps to reproduce?

**Comment by Zetsukaze** [comment]
Have items on the floor tile, then put an empty cart on top of that tile. It seems like some food also disappears. I'm not exactly sure when this occurs, I'm usually crafting or moving stuff around, and soon I notice things underneath the tile with an empty vehicle container disappears. I just had food disappear on me as well.

**Comment by AaronLMorgan** [comment]
OS: Windows 10 64-Bit
Version: 0.C-7240
Tiles or Curses: Tiles
Version String: 0.C-26904-g4aa52a7

Scenario: Challenge-Lab 

Was in a lab that had been substantially chewed out by Giant Naked Mole-Rats. So there was stones, rebar, and other items everywhere. Dumped all of my loot on a single counter top. Included tools, ammo, books, drugs, food, clothing, and spare parts. On two separate occasions I had items disappear from this tile. First it was all of the books that I had collected. Next, I noticed that some of my ammo, food, and drugs had gone missing as well. Both times I had been moving between floors and dumping more items onto this tile. I did not sleep or craft anything during this time.
